runtime/src/bank.rs

```diff
@@ -1509,6 +1509,13 @@ impl Bank {
         )
     }
 
+    fn get_rent_collector_from_parent(
```

Comment:
Maybe just `fn get_rent_collector_from()`, since it's used without a parent in the case of creating a new bank from snapshot fields.